pkgs/development/python-modules/soundcard/default.nix

```
@@ -0,0 +1,49 @@
+{
+  buildPythonPackage,
+  cffi,
+  fetchFromGitHub,
+  lib,
+  libpulseaudio,
+  numpy,
+  setuptools,
+  testers,
+}:
+
+buildPythonPackage rec {
+  pname = "soundcard";
+  version = "0.4.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "bastibe";
+    repo = "soundcard";
+    rev = version;
+    hash = "sha256-d91fhgkjnRFamMIn7hmXuHZt6kGBysAhQq8IsZ/WqHs=";
+  };
+
+  patchPhase = ''
```

This might seem a bit weird, but `postPatch` should be used instead of `patchPhase`, since if you override the default logic of `patchPhase`, the `patches = [ ... ];` list will not work, if you ever want to use `.patch` files for this package in the future.

```suggestion
  postPatch = ''
```